kinto_client/tests/test_cli_utils.py

```diff
@@ -38,6 +38,20 @@ def test_set_parser_server_options_adds_arguments_on_existing_parser(self):
                                    *ALL_PARAMETERS)
         assert len(parser._actions) == 9
 
+    def test_set_parser_server_options_can_ignore_bucket_and_collection(self):
```

Comment:
nit: I didn't realize we had this really long function name. What about renaming it to "add_parser_options" ?